tools/pdf2txt.py

```diff
@@ -116,19 +116,18 @@ def handle_input_variables(
         if converter_params.png_mode:
             converter_params.imagewriter.set_png_mode()
 
-        # Special considerations if the user wants to generate the book chapters as txt.
-        if ConverterParams.get_chapters:
-            chapters = split_by_chapters(filenames[0])
-            write_chapters_to_files(chapters)
-            return
-        convert_from_pdf(filenames, converter_params, outtype, outfile)
+    # Special considerations if the user wants to generate the book chapters as txt.
+    if ConverterParams.get_chapters:
```

Comment:
Same here. Basically any area of the code accessing the member variables of the class should be accessing instance variables instead.

Reply:
fixed